README.md

```diff
@@ -200,9 +200,9 @@ exports.helloCloudEvents = (cloudevent) => {
 };
 ```
 
-To enable automatic unmarshalling, set the function signature type to `cloudevent`
-using a command-line flag or an environment variable. By default, the HTTP
-signature will be used and automatic event unmarshalling will be disabled.
+To enable CloudEvents, set the signature type to `cloudevent`. By default, the HTTP signature will be used and automatic event unmarshalling will be disabled.
+
+Learn how to use CloudEvents in this [guide](https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/docs/cloudevents.md).
```

Comment:
> https://github.com/GoogleCloudPlatform/functions-framework-nodejs/blob/master/docs/cloudevents.md

Usually relative links are better in case we're changing orgs, repos, branches, but linking to a file.

`docs/cloudevents.md`

Reply:
Done!